src/parsers/number.ts

```diff
@@ -97,11 +171,15 @@ export const number: Parser<NumberLike, bigint> = {
    * @returns The decoded value.
    */
   decode({ type, value }: DecodeArgs): bigint {
-    const buffer = value.slice(0, 32);
+    const buffer = value.subarray(0, 32);
```

Comment:
Why are we using `subarray` over `slice` here?

Reply:
`subarray` avoids creating a copy of the data, meaning slightly less memory usage, and slightly higher performance.